cmd/kube-scheduler/app/options/options.go

```diff
@@ -151,12 +151,30 @@ func newDefaultComponentConfig() (*kubeschedulerconfig.KubeSchedulerConfiguratio
 func (o *Options) Flags() (nfs cliflag.NamedFlagSets) {
 	fs := nfs.FlagSet("misc")
 	fs.StringVar(&o.ConfigFile, "config", o.ConfigFile, `The path to the configuration file. The following flags can overwrite fields in this file:
+  --algorithm-provider
+  --policy-config-file
+  --policy-configmap
+  --policy-configmap-namespace
+The following flags will be not effective when set --config , we must set the corresponding parameter value in this file:
```

Comment:
I still prefer we don't mention all of this. It's TMI and prone to become obsolete.

Reply:
I would either list all of the flags that aren't effective, or don't list them at all

Reply:
I prefer we just list the flags that could override

Reply:
@alculquicondor Considering that some information will become obsolete, your concerns are correct :)